tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -1451,6 +1452,22 @@ class DxcCompiler : public IDxcCompiler3,
 // SPIRV change starts
 #ifdef ENABLE_SPIRV_CODEGEN
     compiler.getLangOpts().SPIRV = Opts.GenSPIRV;
+    llvm::Optional<spv_target_env> spirvTargetEnv =
+        spirv::FeatureManager::stringToSpvEnvironment(
+            Opts.SpirvOptions.targetEnv);
+
+    // If we do not have a valid target environment, the error will be handled
+    // later.
```

Comment:
I'm assuming there is already a test to cover this?

Reply:
Yes there is.